capstone_project/src/main/webapp/pages/comments.jsp

```diff
@@ -15,42 +15,63 @@ limitations under the License.
 <%! BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService(); %>
 
 <!DOCTYPE html>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
 <script src="../scripts/comments.js"></script>
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
 <link rel="stylesheet" href="../styles/posts.css">
 <body onload="loadPosts(); loadSchools()">
-  <div id="sort-types">
-    <button onclick="sortPosts('new')">New</button>
-    <button onclick="sortPosts('top')">Top</button>
-    <button onclick="sortPosts('trending')">Trending</button>
+  <div class="row">
+    <div class="col s3" id="sidebar"></div>
+    <div class="col s6">
+      <div class="center" id="sort-types">
+        <button class="btn" onclick="sortPosts('new')">New</button>
+        <button class="btn" onclick="sortPosts('top')">Top</button>
+        <button class="btn" onclick="sortPosts('trending')">Trending</button>
+      </div>
+      <div id="user-posts" data-sort="default"></div>
+    </div>
+    <div class="col s3" id="post-upload-container">
+      <form action="<%= blobstoreService.createUploadUrl("/post-process?file-type=none") %>"
+            enctype="multipart/form-data"
+            id="post-input"
+            method="POST"
+            onsubmit="disableInput(event)">
+        <div class="input-field">
+          <em class="material-icons prefix">title</em>
+          <input id="post-title" name="title" type="text" maxlength="64" required>
+          <label for="post-title">Title</label>
+        </div>
+        <div  class="input-field">
+          <em class="material-icons prefix">school</em>
+          <select name="schools" id="schools" required>
+            <option value="" disabled selected>---</option>
+          </select>
+          <label for="schools">Select a School:</label>
+        </div>
+        <p><a href="maps.html">Don't See Your School?</a></p>
+        <div class="input-field">
+          <em class="material-icons prefix">mode_edit</em>
+          <textarea required
+                    class="materialize-textarea"
+                    id="post-entry"
+                    name="text"></textarea>
+          <label for="post-entry">Text</label>
+        </div>
+        <img alt="" id="image-preview" src="//:0">
```

Comment:
is having this alt field with the empty string necessary? just curious!

Reply:
The Google HTML style guide [recommends](https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback) it if you don't know what the image is. But now that you brought it up, I think I could rename this to Image Preview